tests/test_fips.py

```diff
@@ -105,20 +105,24 @@ def test_openssl_binary(
     img_id = container_runtime.get_image_id_from_stdout(cmd.stdout)
 
     exec_cmd = " ".join(
-        [container_runtime.runner_binary, "run", "--rm"]
+        [container_runtime.runner_binary, "run", "--rm", "--privileged=true"]
```

Comment:
Why do you have to run it privileged?

Reply:
because I'm running mount inside the container